modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/behavior/ListViewBehavior.java

```diff
@@ -340,6 +354,26 @@ public void changed(
         }
     };
 
+    private final InvalidationListener propertiesListener = inv -> {
+        Boolean isComboBoxEditable = (Boolean)getNode().getProperties().get("editableComboBoxEditor");
+        if (isComboBoxEditable != null) {
```

Comment:
Maybe simplify this as follows, to match the earlier logic?

```
if (Boolean.FALSE.equals(getNode().getProperties().get("editableComboBoxEditor"))) {
```

Reply:
With this change, we need to check both FALSE and TRUE.
For a Non ComboBox ListView `getNode().getProperties().get("editableComboBoxEditor")` would be `null`.

```
if (Boolean.FALSE.equals(getNode().getProperties().get("editableComboBoxEditor"))) { add KeyMappings }
else if (Boolean.TRUE.equals(getNode().getProperties().get("editableComboBoxEditor"))) { remove KeyMappings }
else { ListView is not contained by ComboBox }
```